src/js/handler/month/moveGuide.hbs

```diff
@@ -13,7 +13,7 @@
     {{#unless model.isAllDay}}
     <span class="{{CSS_PREFIX}}weekday-schedule-bullet {{CSS_PREFIX}}weekday-schedule-bullet-focused" style="top: {{styles.scheduleBulletTop}}px;"></span>
     {{/unless}}
-    <div class="{{CSS_PREFIX}}month-move-guide {{CSS_PREFIX}}weekday-schedule-title {{#unless model.isAllDay}}{{CSS_PREFIX}}weekday-schedule-title-focused"{{/unless}}>
+    <div class="{{CSS_PREFIX}}month-move-guide {{CSS_PREFIX}}weekday-schedule-title {{#unless model.isAllDay}}{{CSS_PREFIX}}weekday-schedule-title-focused{{/unless}}">
```

Comment:
`{{#unless model.isAllDay}}{{CSS_PREFIX}}weekday-schedule-title-focused{{/unless}}"`
I think you can erase it. It is not used.
The style is being overridden.
https://github.com/nhn/tui.calendar/blob/master/src/css/weekday.styl#L108